src/lib/components/License/LicenseResults.js

```diff
@@ -40,6 +40,7 @@ export const LicenseResults = withState(
                 >
                   <Radio
                     checked={_get(values, 'selectedLicense.title') === title}
+                    className={description ? '' : 'mt-0'}
```

Comment:
just not to have an empty class attribute you can do
```suggestion
                    {...(!description && {className: 'mt-0'})}
```

Reply:
Thanks!